ui/i18n/messages.json

```diff
@@ -81,6 +81,16 @@
   "documentAclTable.until": "Until",
   "documentAclTable.untilMiddle": "until",
   "documentAclTable.userGroup": "User / Group",
+  "documentContent.moveToColdStorage.ask.confirmation":"Send to cold storage ?",
```

Comment:
IIRC this is a french pattern, correct? Having a space between the word and the punctuation 😅
Probably we'll have it on `messages-fr.json`